extensions/amp-instagram/1.0/amp-instagram.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Instagram} from './instagram';
+import {PreactBaseElement} from '../../../src/preact/base-element';
+import {dict} from '../../../src/utils/object';
+import {isLayoutSizeDefined} from '../../../src/layout';
+
+/** @const {string} */
+const TAG = 'amp-instagram';
+
+class AmpInstagram extends PreactBaseElement {
+  /** @override */
+  init() {
+    return dict({
+      'requestResize': (height) => {
+        this.forceChangeHeight(height);
+      },
+    });
+  }
+
+  /** @override */
+  isLayoutSupported(layout) {
+    return isLayoutSizeDefined(layout);
+  }
+}
+
+/** @override */
+AmpInstagram['Component'] = Instagram;
+
+/** @override */
+AmpInstagram['children'] = {};
```

Comment:
Remove `children`? It should be `false`, as by default.
```suggestion
```

Reply:
I added it from Dima's suggestion: https://github.com/ampproject/amphtml/pull/30230#discussion_r492230441

Reply:
@alanorozco I wanted to try it out inside the shadow root right away. If it doesn't work or we hate it, would be very easy to remove.